.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+***Exception or Stack Trace***
+Add the exception log and stack trace if available
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+***Code Snippet***
+Add the code snippet that causes the issue.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Setup (please complete the following information):**
+ - OS: [e.g. iOS]
```

Comment:
In the example, I'd like to see a version number:

`- OS: [e.g. iOS 13]`

Reply:
Agreed, should also do that for IDE as well since it may make a difference between which Xcode they are using.  Same applies to Android/Android Studio